hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2413,6 +2413,9 @@ ce = \case
     -- [ Default, Object, Tag ]
     -- [ False, True, Cond ]
     op "select"
+  DLE_ContractFromAddress _at _addr -> do
+    size <- typeSizeOf $ T_Data (M.fromList [("None", T_Null), ("Some", T_Address)])
```

Comment:
First, you should can cla. Second, there's a function for constructing a Maybe DLType in DLBase